pixel-definitions/pixels/broken_site_reports.json

```diff
@@ -0,0 +1,396 @@
+            },
+            {
+                "key": "blockedTrackers",
+                "description": "Tracker domains blocked on the page. Comma separated list."
+            },
+            {
+                "key": "category",
+                "description": "User chosen breakage category",
+                "enum": [
+                    "",
+                    "comments",
+                    "blocked",
+                    "content",
+                    "dislike",
+                    "empty-spaces",
+                    "images",
+                    "layout",
+                    "links",
+                    "login",
+                    "other",
+                    "paywall",
+                    "shopping",
+                    "videos"
+                ]
+            },
+            {
+                "key": "ctlFacebookLogin",
+                "description": "",
+                "enum": [
+                    "false",
+                    "true"
+                ]
+            },
+            {
+                "key": "ctlFacebookPlaceholderShown",
+                "description": "",
+                "enum": [
+                    "false",
+                    "true"
+                ]
+            },
+            {
+                "key": "ctlYouTube",
+                "description": "",
+                "enum": [
+                    "false",
+                    "true"
+                ]
+            },
+            {
+                "key": "debugFlags",
+                "description": "Comma separated list of C-S-S features active."
+            },
+            {
+                "key": "description",
+                "description": "User-provided description of the breakage."
+            },
+            {
+                "key": "errorDescriptions",
+                "description": "List of main frame status error on page. URL encoded JSON array."
+            },
+            {
+                "key": "httpErrorCodes",
+                "description": "List of HTTP error codes encountered."
+            },
+            {
+                "key": "ignoreRequests",
+                "description": "Tracker domains not blocked due to an exception rule."
+            },
+            {
+                "key": "ignoredByUserRequests",
+                "description": "Tracker domains not blocked due to a user site exception."
+            },
+            {
+                "key": "jsPerformance",
+                "description": ""
+            },
+            {
+                "key": "lastSentDay",
+                "description": "If a report for the same site was previously sent by this user, this indicates the day of that report."
+            },
+            {
+                "key": "locale",
+                "description": "User's browser locale",
+                "pattern": "[a-z][a-z](-[A-Z][A-Z])?"
+            },
+            {
+                "key": "noActionRequests",
+                "description": ""
```

Comment:
Should we use "hostnames of requests that were not present in the blocklist" for the description? (From https://app.asana.com/0/1198207348643509/1200202563872939)